lib/main.dart

```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
 void main() => runApp(const MyApp());
 
 class MyApp extends StatelessWidget {
-
   static Movie marioBros = Movie(
-      title: 'The Super Mario Bros. Movie',
-      originalTitle: 'The Super Mario Bros. Movie',
-      thumbnail: 'images/backdrop_mario_bros_movie.jpg',
-      poster: 'images/poster_mario_bros_movie.jpg',
-      overview: 'While working underground to fix a water main, Brooklyn '
-          'plumbers—and brothers—Mario and Luigi are transported down a '
-          'mysterious pipe and wander into a magical new world. But when the '
-          'brothers are separated, Mario embarks on an epic quest to find '
-          'Luigi.',
-      voteAverage: 7.8,
-      genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
-      countLikes: 0,
+    title: 'The Super Mario Bros. Movie',
+    originalTitle: 'The Super Mario Bros. Movie',
+    thumbnail: 'images/backdrop_mario_bros_movie.jpg',
+    poster: 'images/poster_mario_bros_movie.jpg',
+    overview: 'While working underground to fix a water main, Brooklyn '
+        'plumbers—and brothers—Mario and Luigi are transported down a '
+        'mysterious pipe and wander into a magical new world. But when the '
+        'brothers are separated, Mario embarks on an epic quest to find '
+        'Luigi.',
+    voteAverage: 7.8,
+    genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+    countLikes: 0,
   );
 
   const MyApp({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return  MaterialApp(
+    return MaterialApp(
         home: Scaffold(
-          appBar: AppBar(
-            backgroundColor: Colors.black87,
-            title: Text(marioBros.originalTitle),
-          ),
-          body: SafeArea(
-          top: true,
-          bottom: true,
-          left: true,
-          right: true,
-          child: Center(
-
-            child: ShowMovie(marioBros),
-          )
-          )
-        )
-    );
+            appBar: AppBar(
+              backgroundColor: Colors.black87,
+              title: Text(marioBros.originalTitle),
+            ),
+            body: SafeArea(
+                top: true,
+                bottom: true,
+                left: true,
+                right: true,
```

Comment:
this values of SafeArea are true by default , you can remove them

Reply:
done